packages/spright-components/src/chat/specs/README.md

```diff
@@ -78,15 +71,40 @@ The component also contains the following features:
 1. Displays a vertical scrollbar if there are more messages than fit in the height allocated to the conversation.
 1. Only appearance of its own is to set a background color.
 
+#### Chat input
+
+##### In scope
+
+1. Accepts text input in a text area
+    - the text area height is a single line initially but grows its height to fit the entered text up to its max-height
+    - the text area has configurable placeholder text
+1. Includes a "Send" button for the user to submit the current input text
+    - fires an event containing the current input content and then clears the content and sets keyboard focus back to the input
+    - pressing Enter while the text area has focus will behave the same as clicking "Send"
+    - pressing Shift-Enter will create a newline
+    - the button is disabled and Enter has no effect if the input text is empty
+1. Styling for default, focus, and rollover states
+
+##### Future work
+
+This is included for informational purposes but doesn't yet have a proposed implementation in the Design section below.
+
+1. Includes a "Stop" button for the user to abort an in-progress sent message
```

Comment:
Is there a practical known use-case where we expect this could be useful? I'm guessing this is for use-cases where some pre-processing is done on the message before actually creating a `spright-chat-message` with its contents (I wasn't sure if this was a common/expected scenario)? Or would this be related to the other future work for things like attaching files to cancel an in-progress upload (in which case, I think the verbiage would change to reflect that)?

Reply:
Good question. I don't know for sure, but in the Figma design it looks like it's more about canceling a response to a sent message. I'll update this to be more vague for now. We can revisit if / when we move this from "Future work" to "In scope".